test/functional/feature_notifications.py

```diff
@@ -13,13 +13,19 @@
     connect_nodes,
 )
 
+# Linux allow all characters other than \x00
+# Windows disallow control characters (0-31) and /\?%:|"<>
+FILE_CHAR_START = 32 if os.name == 'nt' else 1
+FILE_CHAR_END = 128
+FILE_CHAR_BLACKLIST = '/\\?%*:|"<>' if os.name == 'nt' else '/'
```

Comment:
Until escaping works on Windows, we should exclude spaces, tabs, and single quotes here too...